mannyhagman/hows-the-weather
# Add sparklines charts for temp, pressure and humidity

#### What's this PR do?
- Adds `react-sparklines` library for chart components
- Calls open weather API and returns data for temp, humidity and pressure 
- Renders data as sparkline charts

#### How should this be manually tested?
- Enter city name in search bar
- Expect to see City name with charts for each category instantly rendered in browser
